backend/backend/settings.py

```
@@ -36,12 +36,13 @@
     "api",
     "rest_framework",
     "corsheaders",
+    "django_extensions" # Can remove this for PRODUCTION
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)